src/main/java/de/tum/in/www1/artemis/service/hestia/ProgrammingExerciseTaskService.java

```diff
@@ -191,6 +196,17 @@ private List<ProgrammingExerciseTask> extractTasks(ProgrammingExercise exercise)
         return tasks;
     }
 
+    /**
+     * Replaces a comma seperated list of test case names with their corresponding ids.
+     * If no matching test case exists (e.g. due to a typo) the test name get kept.
+     */
+    private String extractTestCaseIdsFromNames(String capturedTestCaseNames, Set<ProgrammingExerciseTestCase> testCases) {
+        var testCaseNames = extractTestCaseNames(capturedTestCaseNames);
+
+        return testCaseNames.stream().map(testName -> testCases.stream().filter(tc -> testName.equals(tc.getTestName())).findFirst().map(tc -> tc.getId().toString())
```

Comment:
> or at least being split into multiple lines.

That's not possible since Spotless reverts this change every time. I moved the map() call into its own method.